openr/spark/tests/SparkTest.cpp

```diff
@@ -16,55 +28,57 @@
 #include <openr/common/Constants.h>
 #include <openr/common/NetworkUtil.h>
 #include <openr/common/Util.h>
-#include <openr/config/Config.h>
-#include <openr/config/tests/Utils.h>
+#include <openr/spark/IoProvider.h>
 #include <openr/spark/SparkWrapper.h>
-#include <openr/tests/mocks/MockIoProvider.h>
```

Comment:
We need this. Pls make sure this UT can build and run. 

P.S. it is hard to check this by naked-eyes cuz I can't compile it with my brain. :)